save-preprocessor/src/main/kotlin/org/cqfn/save/preprocessor/controllers/DownloadProject.kt

```diff
@@ -77,4 +97,32 @@ class DownloadProject(val configProperties: ConfigProperties) {
             }
         }
     }
+
+    private fun sendToBackendAndOrchestrator(projectId: Long, path: String) {
+        val execution = Execution(projectId, LocalDateTime.now(), LocalDateTime.now(), ExecutionStatus.PENDING, "1", path)
+        log.debug("Knock-Knock Backend")
+        webClientBackend
+            .post()
+            .uri("/createExecution")
+            .body(BodyInserters.fromValue(execution))
+            .retrieve()
+            .onStatus({status -> status != HttpStatus.OK }) {
+                log.error("Backend internal error")
```

Comment:
Please log status here too
```suggestion
                log.error("Backend internal error: ${it.statusCode()}")
```